runtime/src/main/java/org/corfudb/runtime/view/metadata/Checkpoint.java

```diff
@@ -0,0 +1,14 @@
+package org.corfudb.runtime.view.metadata;
+
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * This class defines metadata associated with a checkpoint
+ */
+public class Checkpoint {
+    private UUID checkpointStream;
+    private long startAddress;
+    private long endAddress;
+    List<Long> checkpointRecords;
```

Comment:
The reason I have it there is to enable bulk read of a checkpoint streams, instead of having to resolve the backpoints from the end address.